pkg/loki/modules.go

```diff
@@ -287,6 +288,21 @@ func (t *Loki) initTableManager() error {
 	}
 
 	t.tableManager.Start()
+
+	t.server.HTTP.Path("/ready").Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// time.Sleep(60 * time.Seconds)
```

Comment:
should we wait here for some time to allow `table-manager` to sync completely first?